memory/stress-ng.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Aneesh Kumar K.V <anesh.kumar@linux.vnet.ibm.com>
+#
+
+import os
+from avocado import Test
+from avocado import main
+from avocado.utils import process, build, archive, distro
+from avocado.utils.software_manager import SoftwareManager
+
+
+def clear_dmesg():
+    process.run("dmesg -c ", sudo=True)
+
+
+def collect_dmesg(object):
+    object.whiteboard = process.system_output("dmesg")
+
+
+class stressng(Test):
+
+    """
+    Stress-ng testsuite
+    :param stressor: Which streess-ng stressor to run (default is "mmapfork")
+    :param ttimeout: Timeout for each run (default 300)
```

Comment:
we have --timeout as a option to the stress-ng . so to differentiate we have test timeout as ttimeout